src/net/IPv6Address.hxx

```diff
@@ -53,7 +53,10 @@ class IPv6Address {
 						   uint16_t c, uint16_t d,
 						   uint16_t e, uint16_t f,
 						   uint16_t g, uint16_t h) noexcept {
-		return {{{
+		return {{
```

Comment:
```suggestion
		return { .s6_addr = {
```

This should be the most portable option as defined in the POSIX specification.

Reply:
This isn't valid C++. Designated initializers are C99 (and will maybe come to C++ with C++20).

Reply:
This is even more portable, and valid C++14: https://github.com/MusicPlayerDaemon/MPD/commit/1fa99da3c254a6a641bdf4eca35d781281c4c268